examples/getting_started/run_getting_started_example.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+# ---------------------------------------------------
+# bash run_getting_started_example.sh 0,1,2,3,4,5,6,7
+# ---------------------------------------------------
+
+#######################
+### INPUT ARGUMENTS ###
+#######################
+if [ -z "$1" ]  # if input argument does not exist
+  then
+    echo "Need to specify the GPU devices as arguments, e.g. bash run_getting_started_example.sh 0,1,2,3,4,5,6,7"
+    exit
+fi
+CUDA_VISIBLE_DEVICES=$1
+
+first_character=${1:0:1}
+if [[ $first_character =~ [^0-7] ]]   # if the first character is not an integer 0-7
+    then
+        echo "First character of specified argument needs to be an integer, e.g. bash run_getting_started_example.sh 0,1,2,3,4,5,6,7"
+        exit
+fi
+
+last_character=${1:0-1}
+if [[ $last_character =~ [^0-7] ]]   # if the first character is not an integer 0-7
```

Comment:
```suggestion
if [[ $last_character =~ [^0-7] ]]   # if the last character is not an integer 0-7
```

Also, shouldn't we check for all of them, rather than only the first and last?

Reply:
Actually, I found another solution now, implemented in efc1629c0c4ef29728dc9cec59e08763274ddf78: The number of GPUs is simply restricted to 2, and the devices are specified like so: `bash run_getting_started_example.sh 0 1`. This allows for simple yet sufficient checking of the input arguments and avoids the aforementioned problems.